cmake/dev-helper.cmake

```diff
@@ -3,46 +3,14 @@ OPTION(WITH_GALERA_DEV "Automatically build & copy & install debug galera librar
 OPTION(WITH_PXB_DEV "Automatically build & copy & install PXB 2.4 and 8.0" OFF)
 
 if(WITH_GALERA_DEV)
-  INCLUDE(ProcessorCount)
-  ProcessorCount(CPU_COUNT)
-
-  IF(CMAKE_BUILD_TYPE_UPPER STREQUAL "DEBUG" OR WITH_DEBUG)
-    SET(GALERA_DEBUG 0)
-  ELSE()
-    SET(GALERA_DEBUG 2)
-  ENDIF()
-
-  MESSAGE(STATUS "Configuring Galera to use debug=${GALERA_DEBUG}")
-
-  # Add a custom target for later refreshes
-  ADD_CUSTOM_TARGET(galera ALL scons -j ${CPU_COUNT} tests=0 debug=${GALERA_DEBUG}
-    COMMAND ${CMAKE_COMMAND} -E copy 
-    "${CMAKE_CURRENT_SOURCE_DIR}/percona-xtradb-cluster-galera/garb/garbd"
-    "${CMAKE_CURRENT_BINARY_DIR}/runtime_output_directory/garbd"
-    COMMAND ${CMAKE_COMMAND} -E copy 
-    "${CMAKE_CURRENT_SOURCE_DIR}/percona-xtradb-cluster-galera/libgalera_smm.so"
-    "${CMAKE_CURRENT_BINARY_DIR}/library_output_directory/libgalera_smm.so"
-    WORKING_DIRECTORY
-    "${CMAKE_CURRENT_SOURCE_DIR}/percona-xtradb-cluster-galera"
-    BYPRODUCTS
-    "${CMAKE_CURRENT_SOURCE_DIR}/percona-xtradb-cluster-galera/garb/garbd"
-    "${CMAKE_CURRENT_SOURCE_DIR}/percona-xtradb-cluster-galera/libgalera_smm.so"
-  )
-
-  INSTALL(PROGRAMS
-    "${CMAKE_CURRENT_SOURCE_DIR}/percona-xtradb-cluster-galera/garb/garbd"
-    DESTINATION bin)
-  INSTALL(PROGRAMS
-    "${CMAKE_CURRENT_SOURCE_DIR}/percona-xtradb-cluster-galera/libgalera_smm.so"
-    DESTINATION lib)
 endif(WITH_GALERA_DEV)
-
+  MESSAGE(WARNING WITH_GALERA_DEV is no longer required, galera is built as part of the normal PXC build)
```

Comment:
Shouldn't it be withing if-endif (WITH_GALERA_DEV) block?. Now if-endif is empty

Reply:
You are right, I didn't notice this when I ran cmake. fixed.